satellite/marketing/marketingweb/server.go

```diff
@@ -28,16 +41,33 @@ type Config struct {
 // Server represents marketing offersweb server
 type Server struct {
 	log *zap.Logger
-
 	config Config
-
 	listener net.Listener
 	server   http.Server
-
+	service  *marketing.Service
 	templateDir string
 }
 
-// commonPages returns templates that are required for everything.
+// Struct used to render each offer table
```

Comment:
https://github.com/storj/storj/wiki/Style#comments I think we are following this convention for commenting

Reply:
done